.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+on:
+  push:
+    branches:
+      - 'release/*'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
📣 I would actually suggest to put the condition only on the release tasks.
So that when preparing a release branch, it is confirmed that it builds successfully. I would also run the tests for a final check.

If you strongly disagree then I'd just remove this `if` and the `branch` filter so that the task is only triggered on the tag as currently, it's strange to use `push` argument `branches` to only do this task on the branch but then filter all the steps out with this `if` condition.

Reply:
I agree with your first suggestion ☺️